Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.2
```

Comment:
Would this support Xcode 11.0? If not then anything blocking us from lowering this to support it?
Xcode 10.x is not / almost not support any more by Apple so it is ok to omit that.

Reply:
I made it 5.1 since that is what comes with 11.0

Reply:
Fixed in the new PR
https://github.com/OneSignal/OneSignal-iOS-SDK/pull/712/files#diff-37ca2dd15ca0f6b1b49e78db084ef5b9R1